media-nep/src/main/java/nl/vpro/nep/service/impl/NEPItemizeServiceImpl.java

```diff
@@ -217,6 +214,28 @@ public String getMidItemizerString() {
         return itemizeMidUrl;
     }
 
+    @Override
+    public ItemizerStatusResponse getItemizerJobStatus(String jobId) {
+        String jobs = itemizeLiveUrl + "/api/itemizer/jobs/" + jobId + "/status";
+        HttpGet get = new HttpGet(jobs);
+        authenticate(get, itemizeLiveKey);
+        HttpClientContext clientContext = HttpClientContext.create();
+
+        try (CloseableHttpResponse execute = httpClient.execute(get, clientContext)) {
+            if (execute.getStatusLine().getStatusCode() == 200) {
+                ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+                IOUtils.copy(execute.getEntity().getContent(), outputStream);
+                return Jackson2Mapper.getLenientInstance().readValue(outputStream.toByteArray(), ItemizerStatusResponse.class);
+            } else {
+                StringWriter result = new StringWriter();
+                IOUtils.copy(execute.getEntity().getContent(), result, Charset.defaultCharset());
+                throw new RuntimeException(result.toString());
```

Comment:
SonarCube plugin zal waarschuwing tonen bij new RuntimeExc..

Reply:
Dit stukje is sowieso eigenlijk nog work in progress. Volgens de documentatie zou er ook een status 404 uit moeten kunnen komen en zo. Maar dat gebeurt niet. 

Ik heb er voor nu NEPException van gemaakt, als we meer weten hoe deze exceptionele sistuatie moet gaan, dan kijk ik verder.
Op zich zou error-situatie ook een andere json teruggeven, die wil je dan misschien ook unmarshallen in de excceptie of zo.